src/load

```diff
@@ -346,17 +392,46 @@ def save_capec_data(appctx, args, db_records):
 
         db_insert_progress.close()
 
+# ------------------------------------------------------------------------------
+# Function to save EPSS data to the database
+def save_epss_data(appctx, args, epss_records):
+    from sqlalchemy.orm import load_only
+
+    with appctx.db as session:
+
+        db_insert_progress_name = 'Vuln db update'
+        bar_format = '{n_fmt}/{total} {l_bar}{bar}| ({elapsed}/{remaining})'
+        db_insert_progress = tqdm(total=len(epss_records), bar_format=f'{db_insert_progress_name:<20} {bar_format}',ascii=True)
+
+        for batch in [epss_records[i:i + BATCH_SIZE] for i in range(0, len(epss_records), BATCH_SIZE)]:
+            vuln_query= session.query(Vuln).filter(Vuln.vuln_id.in_([item['cve'] for item in batch])).options(load_only(Vuln.data))
+
+            # Iterate over the fetched records
+            for vuln_record in vuln_query.all():
+                # Find the corresponding record in the batch
+                epss_desired_row = next((row for row in batch if row['cve'] == vuln_record.vuln_id), None)
```

Comment:
See prev comment, the search for the epss score info would be based on the CVE-id.